.github/workflows/ci.yml

```diff
@@ -61,7 +61,9 @@ jobs:
 
       - name: Cargo check
         uses: actions-rs/cargo@v1
-        continue-on-error: ${{ matrix.config.toolchain != 'stable' }}
+        env:
```

Comment:
Are you reverting this from your previous change?

Reply:
Yes, I'm fixing this because the code should just build with Beta IMO

Reply:
Okay. Maybe split this into a separate commit to clarify this (like you did for the previous PR)?

Reply:
👍 reverted